spec/features/unlinking_a_defendant_from_maat_v2_spec.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+
+require 'court_data_adaptor'
+
+RSpec.feature 'Unlinking a defendant from MAAT', type: :feature, stub_unlink_v2: true do
+  let(:defendant_nino_from_fixture) { 'JC123456A' }
+  let(:case_urn) { 'TEST12345' }
+  let(:api_url_v2) { CdApi::BaseModel.site }
+  let(:api_request_path) { "#{api_url_v2}laa_references/#{defendant_id}/" }
+  let(:defendant_asn_from_fixture) { '0TSQT1LMI7CR' }
```

Comment:
is this needed:

- defendant_nino_from_fixture
- defendant_asn_from_fixture
- defendant_fixture
- defendant_by_id_fixture

Reply:
These are variables used in the webmock stubs - removing them would break the test